modules/trusted-firmware-m/Kconfig

```diff
@@ -71,6 +71,40 @@ config TFM_ITS_ENCRYPTED
 	help
 	  Enables authenticated encryption for PSA Internal Trusted Storage files
 
+config TFM_HW_INIT_RESET_ON_BOOT
+	bool "Initialize internal architecture state at boot"
+	depends on ARCH_SUPPORTS_ARCH_HW_INIT
+	help
+	  This option instructs TF-M to force the initialization
+	  of the internal architectural state (for example ARCH-level
+	  HW registers and system control blocks) during boot to
+	  the reset values as specified by the corresponding
+	  architecture manual. The option is useful when the TF-M
+	  firmware image is chain-loaded, for example, by a debugger
```

Comment:
NITS: The other documentated topics call it "the TF-M image" whentalking about the SPE